### DISCUSSION TITLE
Raspberry Pi Supported?

### DISCUSSION THREAD

**Discussion by laGameTV** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#492] [Q&A]
Im a newbie to python and i tried cloning the repo on my pi 4 but this doesnt really work.

**Comment by Tkd-Alex** [comment]
Describe better your problem.
The script It's fully operational on Raspberry

**Reply by laGameTV** [reply]
https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/493
